merlin-server/sql/merlin/tables/activity_directive_metadata_schema.sql

```diff
@@ -0,0 +1,31 @@
+create table activity_directive_metadata_schema (
+  key text not null primary key,
```

Comment:
If I understand correctly, we want these schema to be defined per mission model - I think we want to add a `model_id` column to this table, and include it in the primary key.

Reply:
The design here was one schema per deployment, not per mission model. I'll let @patkenneally or @bashbash change their minds on that if they want though.

Reply:
I see. I do remember that discussion now that you mention it. Resolving this conversation, then.